fairlearn/metrics/_metric_frame.py

```diff
@@ -189,6 +189,32 @@ class MetricFrame:
 
         .. deprecated:: 0.7.0
             `metric` will be removed in version 0.10.0, use `metrics` instead.
+
+    Examples
+    --------
+    >>> from fairlearn.metrics import MetricFrame, selection_rate
```

Comment:
Is this code actually executed? As in, if you change a number will that break the documentation build? If so, that's exactly what I'd want here :-)

Reply:
Sadly not, wihtout explicitly writing the output in the docstring there would be no output shown. However, I am not sure if it is possible to have code executed since it is written in the docstring of the class to be shown in the API reference automatically. I looked at the source code of numpy, pandas and scipy and this is how they did it.

Reply:
That is perfect! Thanks for checking!